src/components/Article-comments.jsx

```diff
@@ -0,0 +1,28 @@
+import { useState, useEffect } from "react"
+import CommentCard from "./CommentCard"
+
+const ArticleCommentsSection = (props) => {
+
+    const currentArticleComments = props.currentArticleComments
+    const setCurrentArticleComments = props.setCurrentArticleComments
+
+    const [currentComments, setCurrentComments] = useState(currentArticleComments)
```

Comment:
we dont need to save the comments to a state here - we can directly just map through the currentArticleComments. otherwise consider moving your getCommentsById invocation to this component instead of making it happen in the single article page component - it then doesn't depend on the article to load